crates/loam-cli/src/commands/build/build_clients.rs

```diff
@@ -1,23 +1,23 @@
 #![allow(clippy::struct_excessive_bools)]
+use crate::commands::build::environments_toml;
```

Comment:
Could this be `envs_toml`? Seems long if you are going to use it as a prefix.

Reply:
Sure, even better, let's drop that `s`. Renamed to `env_toml`